.gitignore

```
@@ -29,3 +29,8 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+/public/packs
+/public/packs-test
+/node_modules
+yarn-debug.log*
+.yarn-integrity
```

Can you try this as well?
```suggestion
.yarn-integrity
**/.DS_Store
```

---

Added the **/.DS_Store line to the .gitignore file.